sos/policies/distros/redflag.py

```diff
@@ -0,0 +1,50 @@
+# This Python file uses the following encoding: utf-8
+
+# 版权所有（c）Yang Qing <yangqing@redflag-os.com>
+#
+# 此文件是sos项目的一部分：https://github.com/sosreport/sos
+#
+# 该受版权保护的材料可供任何希望使用的人使用，
+# 根据以下条款和条件对其进行修改、复制或重新分发
+# GNU通用公共许可证的第2版。
+#
+# 有关详细信息，请参阅源发行版中的LICENSE文件。
+
+from sos.policies.distros.redhat import RedHatPolicy, OS_RELEASE
+import os
+
+
+class RedFlagPolicy(RedHatPolicy):
+    distro = "RedFlag"
+    vendor = "RedFlag Software"
+    vendor_urls = [('Distribution Website', 'http://www.redflag-os.com')]
```

Comment:
My original point was a nitpick "can you provide rather a link to https pages, please?" but it seems the project runs (untrusted) http pages only..? (then this line would be fully OK).